### DISCUSSION TITLE
Is this bannable?

### DISCUSSION THREAD

**Discussion by minecraft-go-new** [JustOptimize/return-ShowHiddenChannels#24] [Q&A] [answered]
Can I randomly get banned because of this plugin and what % that this will happened?

**Comment by JustOptimize** [comment, marked as answer]
Betterdiscord is itself bannable being a modification of the client thus against the TOS of discord but I haven't seen anyone get banned for it. This plugin does not increase the risk of getting banned but as a precaution avoid sending around screenshots where you see hidden channels or you are using a theme
